src/cs/lib/msquic_generated.cs

```diff
@@ -0,0 +1,2243 @@
+    {
+        QUIC_STREAM_SHUTDOWN_FLAG_NONE = 0x0000,
+        QUIC_STREAM_SHUTDOWN_FLAG_GRACEFUL = 0x0001,
+        QUIC_STREAM_SHUTDOWN_FLAG_ABORT_SEND = 0x0002,
+        QUIC_STREAM_SHUTDOWN_FLAG_ABORT_RECEIVE = 0x0004,
+        QUIC_STREAM_SHUTDOWN_FLAG_ABORT = 0x0006,
+        QUIC_STREAM_SHUTDOWN_FLAG_IMMEDIATE = 0x0008,
+    }
+
+    public enum QUIC_RECEIVE_FLAGS
+    {
+        QUIC_RECEIVE_FLAG_NONE = 0x0000,
+        QUIC_RECEIVE_FLAG_0_RTT = 0x0001,
+        QUIC_RECEIVE_FLAG_FIN = 0x0002,
+    }
+
+    public enum QUIC_SEND_FLAGS
+    {
+        QUIC_SEND_FLAG_NONE = 0x0000,
+        QUIC_SEND_FLAG_ALLOW_0_RTT = 0x0001,
+        QUIC_SEND_FLAG_START = 0x0002,
+        QUIC_SEND_FLAG_FIN = 0x0004,
+        QUIC_SEND_FLAG_DGRAM_PRIORITY = 0x0008,
+        QUIC_SEND_FLAG_DELAY_SEND = 0x0010,
+    }
+
+    public enum QUIC_DATAGRAM_SEND_STATE
+    {
+        QUIC_DATAGRAM_SEND_SENT,
+        QUIC_DATAGRAM_SEND_LOST_SUSPECT,
+        QUIC_DATAGRAM_SEND_LOST_DISCARDED,
+        QUIC_DATAGRAM_SEND_ACKNOWLEDGED,
+        QUIC_DATAGRAM_SEND_ACKNOWLEDGED_SPURIOUS,
+        QUIC_DATAGRAM_SEND_CANCELED,
+    }
+
+    public unsafe partial struct QUIC_REGISTRATION_CONFIG
+    {
+        [NativeTypeName("const char *")]
+        public byte* AppName;
+
+        public QUIC_EXECUTION_PROFILE ExecutionProfile;
+    }
+
+    public unsafe partial struct QUIC_CERTIFICATE_HASH
+    {
+        [NativeTypeName("uint8_t [20]")]
+        public fixed byte ShaHash[20];
+    }
+
+    public unsafe partial struct QUIC_CERTIFICATE_HASH_STORE
+    {
+        public QUIC_CERTIFICATE_HASH_STORE_FLAGS Flags;
+
+        [NativeTypeName("uint8_t [20]")]
+        public fixed byte ShaHash[20];
+
+        [NativeTypeName("char [128]")]
+        public fixed byte StoreName[128];
+    }
+
+    public unsafe partial struct QUIC_CERTIFICATE_FILE
+    {
+        [NativeTypeName("const char *")]
+        public byte* PrivateKeyFile;
+
+        [NativeTypeName("const char *")]
+        public byte* CertificateFile;
+    }
+
+    public unsafe partial struct QUIC_CERTIFICATE_FILE_PROTECTED
+    {
+        [NativeTypeName("const char *")]
+        public byte* PrivateKeyFile;
+
+        [NativeTypeName("const char *")]
+        public byte* CertificateFile;
+
+        [NativeTypeName("const char *")]
+        public byte* PrivateKeyPassword;
+    }
+
+    public unsafe partial struct QUIC_CERTIFICATE_PKCS12
+    {
+        [NativeTypeName("const uint8_t *")]
+        public byte* Asn1Blob;
+
+        [NativeTypeName("uint32_t")]
+        public uint Asn1BlobLength;
+
+        [NativeTypeName("const char *")]
+        public byte* PrivateKeyPassword;
+    }
+
+    public unsafe partial struct QUIC_CREDENTIAL_CONFIG
+    {
+        public QUIC_CREDENTIAL_TYPE Type;
+
+        public QUIC_CREDENTIAL_FLAGS Flags;
+
+        [NativeTypeName("QUIC_CREDENTIAL_CONFIG::(anonymous union at C:/Users/thhous/Documents/GitHub/msquic/src/inc/msquic.h:274:5)")]
```

Comment:
Don't check in `C:/Users/thhous/Documents/GitHub`

Reply:
Whoops yeah. I'll fix that. Thats part of the generator, and it doesn't currently have a way to bypass that. I'll make an issue on their end.